tracing-undertow/src/main/java/com/palantir/tracing/undertow/UndertowTracing.java

```diff
@@ -101,14 +103,28 @@ public void exchangeEvent(HttpServerExchange exchange, NextListener nextListener
             try {
                 DetachedSpan detachedSpan = exchange.getAttachment(REQUEST_SPAN);
                 if (detachedSpan != null) {
-                    detachedSpan.complete();
+                    detachedSpan.complete(statusMetadata(exchange.getStatusCode()));
                 }
             } finally {
                 nextListener.proceed();
             }
         }
     }
 
+    private static final ImmutableMap<String, String> METADATA_200 = ImmutableMap.of("status", "200");
```

Comment:
Nope, I just figured it would be helpful to tag requests with a status code since it allows us to quickly differentiate successes from failures and demonstrate how the tagging feature works. I've updated the implementation a tad since this comment.